src/Services/Processor/Substrate/ExtrinsicProcessor.php

```diff
@@ -50,9 +50,13 @@ public function run(): array
 
     protected function processExtrinsic(PolkadartExtrinsic $extrinsic, int $index)
     {
+        if (empty($extrinsic->signer)) {
+            return;
+        }
+
         $transaction = Transaction::where([
             'transaction_chain_hash' => $extrinsic->hash,
-            'wallet_public_key' => SS58Address::getPublicKey($extrinsic->signer),
+            'wallet_public_key' => HexConverter::prefix(SS58Address::getPublicKey($extrinsic->signer)),
```

Comment:
<img width="818" alt="Screenshot 2023-11-21 at 1 41 18 PM" src="https://github.com/enjin/platform-core/assets/114082473/3e4a8542-8345-4d96-a2d3-df51a8627b9a">

The function already have prefix inside

Reply:
Awesome, so maybe we only need the empty and the wallet daemon one.

Reply:
Yep, confirmed.